app/models/lockstep/user.rb

```diff
@@ -3,6 +3,15 @@ class Lockstep::User < Lockstep::ApiRecord
   self.id_ref = "user_id"
   load_schema(Schema::UserAccount)
 
+  def self.invite(email_addresses)
```

Comment:
This method should be defined after the `belongs_to :account` code

Reply:
Added [here](https://github.com/Lockstep-Network/lockstep-sdk-ruby-rails/pull/44/commits/dcc473c36d201f48d2d878ce0aa2ad6caf42ff01)